_includes/notebook_github_link

```diff
@@ -0,0 +1,4 @@
+<a class="btn btn-sm text-gray-dark" href="https://github.com/{{site.github_username}}/{{site.github_repo}}/tree/{{page.branch | default: "master" }}/{{page.nb_path}}" role="button">
```

Comment:
```suggestion
<a class="btn btn-sm text-gray-dark" href="{{ site.url }}{{ site.baseurl }}/tree/{{page.branch | default: "master" }}/{{page.nb_path}}" role="button">
```
I believe this will work per my earlier comment re: the special plugin already built in